scrapers_next/va/committees.py

```diff
@@ -0,0 +1,149 @@
+from spatula import URL, CSS, HtmlListPage, HtmlPage, SelectorError, SkipItem
+from openstates.models import ScrapeCommittee
+import time
+
+# VA lists the full names of committee members on individual separate pages
+# MemberDetail grabs a member's full name from their specific page
+
+
+class MemberDetail(HtmlPage):
+    def process_page(self):
+        com = self.input[0]
```

Comment:
Might be worth handling when `input` _isn't_ a >= 2 item list.

Reply:
Ok! I'm going to throw a ValueError if there isn't a 2 item list. This part of the code should only get called with a 2 item list (line 68) so I feel like if the two items don't make it into this portion that's...not good.